lib/document-to-html.js

```diff
@@ -358,7 +374,7 @@ function DocumentConversion(options, comments) {
     function createDeferedChart(element, messages, options){
         return element.read().then(
             function (value) {
-                return Html.freshElement('pwl:chart', null, [Html.text(JSON.stringify(value))]);
+                return Html.freshElement('pwl:chart', null, [Html.text('Chart')]);//[Html.text(JSON.stringify(value))]);
```

Comment:
Could remove the end comment?

Reply:
The end comment will potentially be restored when we know what we are doing with it